src/integration-test/java/com/commercetools/sync/inventories/InventoryIntegrationTestUtils.java

```diff
@@ -0,0 +1,186 @@
+     * Creates inventory entry of values: SKU_1, QUANTITY_ON_STOCK_1, EXPECTED_DELIVERY_1, RESTOCKABLE_IN_DAYS_1.
+     * Creates inventory entry of values: SKU_1, QUANTITY_ON_STOCK_2, EXPECTED_DELIVERY_2, RESTOCKABLE_IN_DAYS_2 and
+     * reference to firstly created supply channel.
+     * Creates inventory entry of values: SKU_1, QUANTITY_ON_STOCK_2, EXPECTED_DELIVERY_2, RESTOCKABLE_IN_DAYS_2 and
+     * reference to secondly created supply channel.
+     */
+    static void populateSourceProject() {
+        final ChannelDraft channelDraft1 = ChannelDraft.of(SUPPLY_CHANNEL_KEY_1)
+            .withRoles(ChannelRole.INVENTORY_SUPPLY);
+        final ChannelDraft channelDraft2 = ChannelDraft.of(SUPPLY_CHANNEL_KEY_2)
+            .withRoles(ChannelRole.INVENTORY_SUPPLY);
+
+        final String channelId1 = CTP_SOURCE_CLIENT
+            .execute(ChannelCreateCommand.of(channelDraft1))
+            .toCompletableFuture()
+            .join()
+            .getId();
+        final String channelId2 = CTP_SOURCE_CLIENT
+            .execute(ChannelCreateCommand.of(channelDraft2))
+            .toCompletableFuture()
+            .join()
+            .getId();
+
+        final Reference<Channel> supplyChannelReference1 = Channel.referenceOfId(channelId1);
+        final Reference<Channel> supplyChannelReference2 = Channel.referenceOfId(channelId2);
+
+        final InventoryEntryDraft draft1 = InventoryEntryDraftBuilder.of(SKU_1, QUANTITY_ON_STOCK_1,
+            EXPECTED_DELIVERY_1, RESTOCKABLE_IN_DAYS_1, null).build();
+        final InventoryEntryDraft draft2 = InventoryEntryDraftBuilder.of(SKU_1, QUANTITY_ON_STOCK_2,
+            EXPECTED_DELIVERY_2, RESTOCKABLE_IN_DAYS_2, supplyChannelReference1).build();
+        final InventoryEntryDraft draft3 = InventoryEntryDraftBuilder.of(SKU_1, QUANTITY_ON_STOCK_2,
+            EXPECTED_DELIVERY_2, RESTOCKABLE_IN_DAYS_2, supplyChannelReference2).build();
+
+        CTP_SOURCE_CLIENT.execute(InventoryEntryCreateCommand.of(draft1)).toCompletableFuture().join();
+        CTP_SOURCE_CLIENT.execute(InventoryEntryCreateCommand.of(draft2)).toCompletableFuture().join();
+        CTP_SOURCE_CLIENT.execute(InventoryEntryCreateCommand.of(draft3)).toCompletableFuture().join();
+
+    }
+
+    /**
+     * Populate target CTP project.
+     * Creates supply channel of key SUPPLY_CHANNEL_KEY_1.
+     * Creates inventory entry of values: SKU_1, QUANTITY_ON_STOCK_1, EXPECTED_DELIVERY_1, RESTOCKABLE_IN_DAYS_1.
+     * Creates inventory entry of values: SKU_1, QUANTITY_ON_STOCK_1, EXPECTED_DELIVERY_1, RESTOCKABLE_IN_DAYS_1 and
+     * reference to supply channel created before.
+     */
+    static void populateTargetProject() {
+        final ChannelDraft channelDraft = ChannelDraft.of(SUPPLY_CHANNEL_KEY_1)
+            .withRoles(ChannelRole.INVENTORY_SUPPLY);
+        final String channelId = CTP_TARGET_CLIENT
+            .execute(ChannelCreateCommand.of(channelDraft))
```

Comment:
Bad idea IMHO. There is an integration test, that checks if service works fine. So using that service would be like: a service would be tested by test code which depends on that very same service. In above snippet to populate projects I use only JDK-SDK code what is... more adequat. What's more, services can be changed what may affect tests.